### ISSUE TITLE
Chip group has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly#2528]
- IDs used in ARIA and labels must be unique (seems to be issue with expanded chips)
- id attribute value must be unique (issue with headings)

**Comment by jessiehuff** [comment]
This issue appears to be solved. 🙂
